support/cas-server-support-oidc/src/main/java/org/apereo/cas/oidc/jwks/OidcJsonWebKeystoreGeneratorService.java

```diff
@@ -60,9 +61,16 @@ protected void generate(final Resource file, final int bits) {
             val rsaJsonWebKey = RsaJwkGenerator.generateJwk(bits);
             val jsonWebKeySet = new JsonWebKeySet(rsaJsonWebKey);
             val data = jsonWebKeySet.toJson(JsonWebKey.OutputControlLevel.INCLUDE_PRIVATE);
-            val location = file instanceof FileSystemResource
-                ? FileSystemResource.class.cast(file).getFile()
-                : DEFAULT_JWKS_LOCATION;
+            File location;
```

Comment:
How am I supposed to write this declaration?

Checkstyle want's me to use `lombok.val` or `var`, but this cannot be used here:

```
/Users/larsgrefer/git/apereo/cas/support/cas-server-support-oidc/src/main/java/org/apereo/cas/oidc/jwks/OidcJsonWebKeystoreGeneratorService.java:64: error: cannot infer type for local variable location
            var location;
                ^
  (cannot use 'var' on variable without initializer)
```

Reply:
Refactor into a separate and smaller method and return the result after each check.